kubectx

```diff
@@ -22,7 +22,10 @@ set -eou pipefail
 IFS=$'\n\t'
 
 SELF_CMD="$0"
-KUBECTX="${XDG_CACHE_HOME:-$HOME/.kube}/kubectx"
+KUBECTX_CACHE_DIR="${XDG_CACHE_HOME:-$HOME/.kube}/cache/kubectx"
+[[ ! -d ${KUBECTX_CACHE_DIR} ]] && mkdir -p ${KUBECTX_CACHE_DIR}
+KUBECTX_CACHE="${KUBECTX_CACHE_DIR}/kubectx"
```

Comment:
I'd actually rename that to `PREVIOUS_CONTEXT_CACHE`, since it's really used as a storage for previous context.